collections/_article/dev-snapshot-godot-4-3-dev-1.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Dev snapshot: Godot 4.3 dev 1"
+excerpt: "We're ending the year with an early sneak peak at the upcoming Godot 4.3 release with its first dev snapshot!"
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-3-dev-1.webp
+image_caption_title: "Yeetus"
+image_caption_description: "A game by YarnCat Games"
+date: 2023-12-21 17:00:00
+---
+
+This was an exciting year for Godot! Our contributors managed to deliver massive, feature-packed releases 3 times over our previous norm. We can't wait to see and share what the international community of passionate game and game engine developers can create for you in the year 2024. Actually, they have already started their work on the upcoming Godot 4.3 release.
+
+And it's a holiday season, so we cannot leave you without a small gift as we wrap up the production and plan our leisure time. Please, enjoy the first dev snapshot of Godot 4.3, which includes quite a few noteworthy improvements for you to play with. It also contains a handful of fixes, including pretty much everything [published last week in Godot 4.2.1](/article/maintenance-release-godot-4-2-1/).
```

Comment:
```suggestion
And it's holiday season, so we cannot leave you without a small gift as we wrap up the production and plan our leisure time. Please, enjoy the first dev snapshot of Godot 4.3, which includes quite a few noteworthy improvements for you to play with. It also contains a handful of fixes, including pretty much everything [published last week in Godot 4.2.1](/article/maintenance-release-godot-4-2-1/).
```

Reply:
Well, there may be other holiday seasons throughout the year :P But I'll take it.